app/models/case_log.rb

```diff
@@ -210,7 +210,7 @@ def tshortfall_unknown?
   end
 
   def is_fixed_term_tenancy?
-    [4, 6].include?(tenancy)
+    [4, 6, 7].include?(tenancy)
```

Comment:
Does this affect any production logs?

Reply:
This is mostly used in validations. As 7 is a new value, the only risk is that some logs could be now invalid.

Reply:
Can we check before hand if any production logs are affected?

Reply:
I think I am going to remove the value, we are not checking for fixed terms for option 2 (which was lifetime). We have 49 records with option 7 (assured lifetime) and no tenancylength.